src/lib/urn.ts

```diff
@@ -89,6 +89,10 @@ export function buildCatalystItemURN(contractAddress: string, tokenId: string):
   return `urn:decentraland:${getNetworkURNProtocol(Network.MATIC)}:collections-v2:${contractAddress}:${tokenId}`
 }
 
+export function buildDefaultCatalystCollectionURN() {
+  return `urn:decentraland:${getNetworkURNProtocol(Network.MATIC)}:collections-v2:0x0000000000000000000000000000000000000000`
+}
```

Comment:
Question, would it make sense to generalize `buildCatalystItemURN`?

maybe:

```ts

export function buildCatalystItemURN(contractAddress: string, tokenId?: string): URN {
 let urn = `urn:decentraland:${getNetworkURNProtocol(Network.MATIC)}:collections-v2:${contractAddress}`
  if(tokenId) {
    urn += `:${tokenId}`  
  }
  return urn
}

export function buildDefaultCatalystCollectionURN() {
  return buildCatalystItemURN(Address.ZERO.toString())
}

Reply:
I wouldn't generalize it because the catalystItemURN needs to have a token id to be a valid item URN. If we make it optional, we'll open the door to people using it incorrectly.

Reply:
Gotcha, thought that'd be the case